requirements.txt

```diff
@@ -1,18 +1,17 @@
-Django>=1.11,<2.3
-django-auth-ldap<2.2
-django-compressor==2.4
-django-crispy-forms>=1.6,<1.11.3
-django-debug-toolbar<3.0.0
-django-wkhtmltopdf==3.3.0
-mysqlclient<2.0.4
-eventlog==0.11.0
-icalendar==4.0.7
-jsonfield<3.1.0
+Django==3.2.14
```

Comment:
<3.3 and maybe >3.0?

Reply:
Technically, you need to pin your dependencies in requirements.txt to an exact version to avoid inconsistency between development and production. Maybe it's an idea to use [Poetry](https://python-poetry.org/) to make dependency management less painful.

Reply:
Agreed, although the subversions of Django are always compatible and only bring bug and security fixes. As we receive an email notification for every update, we update all environments in time. Poetry looks interesting, although if we go for that, we probably would want to implement that for all repos at the same time.